### DISCUSSION TITLE
Standup (Wednesday, November 23th)

### DISCUSSION THREAD

**Discussion by DavBot02** [BBGTC/food-bank-app#68] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Opened #67 (not ready to merge yet)

**What are your priorities for the next 24 hours?**:
 - Merge #67 
 - Work on Profile View

**Blockers**:
 - Haven't set env (just realized that was the problem)

**Shoutouts**:
 - @antonieto for presenting a cool idea

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Nothing tbh

**What are your priorities for the next 24 hours?**:
 -  Review PRs 
 -  Work on News View and finished it

**Blockers**:
 - Elective course homework is taking a lot of time

**Shoutouts**:
 - @antonieto for taking a selfie with some Buchonas at Bossé

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished the main views in the admin client app
 - Sketched a flow diagram with my plans to implementing the auth for the client

**What are your priorities for the next 24 hours?**:
 - Finish the implementation

**Blockers**:
 - Elective assignments

**Shoutouts**:
 - @DavBot02 for being a nice travelling partner
